assigner/commands/status.py

```diff
@@ -85,16 +85,15 @@ def status(conf, args):
             if branches:
                 row[5] = ", ".join([b["name"] for b in branches])
 
-            commits = repo.list_commits()
+            head = repo.get_last_HEAD_commit()
 
-            if commits:
-                head = commits[0]
+            if head:
                 row[6] = head["short_id"]
                 row[7] = head["author_name"]
                 created_at = head["created_at"]
                 # Fix UTC offset format in GitLab's datetime
                 created_at = created_at[:-6] + created_at[-6:].replace(':', '')
-                row[8] = datetime.strptime(created_at, "%Y-%m-%dT%H:%M:%S.%f%z")
+                row[8] = datetime.strptime(created_at, "%Y-%m-%dT%H:%M:%S.%f%z").astimezone().strftime("%c")
```

Comment:
line length

Reply:
What's the pythonic way to wrap this? I want to do something like

```
datetime
.strptime()
.astimezone()
.strftime()
```

but I'd need line continuations to make that parse right.

Reply:
```python
row[8] = datetime.strptime(
    created_at, "%Y-%m-%dT%H:%M:%S.%f%z"
).astimezone().strftime("%c") 
```
Is my personal favorite. I don't think leading with periods like that is common. I also would like that, but that's more of a ruby/js thing in my experience.